src/EmployerFeedback/SFA.DAS.EmployerFeedback/Application/Queries/GetTrainingProviderSearch/GetTrainingProviderSearchQueryHandler.cs

```diff
@@ -118,7 +118,7 @@ private async Task<List<Provider>> GetActiveMainProvidersAsync(CancellationToken
             try
             {
                 var roatpProvidersResponse = await _roatpV2TrainingProviderService.GetProviders(true);
-                var providers = roatpProvidersResponse?.RegisteredProviders?.Where(p => p.ProviderTypeId == 1).ToList() ?? [];
+                var providers = roatpProvidersResponse?.RegisteredProviders?.Where(p => p.ProviderTypeId == 1 && (p.StatusId == 1 || p.StatusId == 2)).ToList() ?? [];
```

Comment:
I know I used a Magic number 1 here - but it's all two obvious that you have added two more magic numbers - add some constants